libraries/vircadia-client/src/client.cpp

```diff
@@ -0,0 +1,154 @@
+//
+//  client.cpp
+//  libraries/client/src
+//
+//  Created by Nshan G. on 15 March 2022.
+//  Copyright 2022 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+#include "client.h"
+
+#include <list>
+
+#include <nlohmann/json.hpp>
+
+#include "version.h"
+#include "internal/Context.h"
+
+namespace vircadia { namespace client
+{
```

Comment:
```suggestion
namespace vircadia { namespace client {
```

Reply:
This code is no longer there in the next PR, so will leave it as is to avoid conflicts.